src/components/DataTable/table.helpers.js

```diff
@@ -18,7 +19,7 @@ export function mapper(key, obj) {
   return obj[key]
 }
 
-export function sortBy(data, fields, by, order) {
+export function sortBy(data, fields, by, order, sortOptions) {
```

Comment:
options params are usually objects. you are using an array in this case. can you paste and example usage of this?

Reply:
Yes, sure.

I already included it in the description.

```
[{
  fieldName: string, // the field's name
  ignoreCase: boolean, // whether casing should be ignored, default: false
}]
```

Usage example:

```
<DataTable
  ...
  :sortOptions="[{ fieldName: 'product.Name', ignoreCase: true }]"
  ...
>
...
</DataTable>
```

Reply:
```js
<DataTable 
  ... 
  :sortModifiers="{ 'product.Name': val => val.toLowerCase() }" 
  ... 
> 
  ... 
</DataTable>
```

and in helpers:
```
    const sortModifier = sortModifiers && sortModifiers[by]

    if (sortModifier)
      return orderBy(data, [(item) => sortModifier(get(item, by))], [order])
    else return orderBy(data, [by], [order])
```

Reply:
i think you 'll have much more freedom this way

Reply:
Nice suggestion!